pandas-stubs/core/frame.pyi

```diff
@@ -272,13 +272,22 @@ class DataFrame(NDFrame, OpsMixin):
     def dot(self, other: Series) -> Series: ...
     def __matmul__(self, other): ...
     def __rmatmul__(self, other): ...
+    @overload
+    @classmethod
+    def from_dict(
+        cls,
+        data: dict[Any, Any],
+        orient: Literal["index"] = ...,
```

Comment:
```suggestion
        orient: Literal["index"],
```
If you use `...`, then it allows `orient` to not be specified, so without it, that means this overload will only apply if `orient=="index"`

Reply:
corrected in bc05adef9112340636940500d5a0b4ab66f80c4d